### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by transcental** [TheIoLang/theiolang.github.io#1]
Documentation for IO commands and functions.
This will come under `https://theiolang.github.io/docs` and files will be located in the `docs/*` folder.

_@JBYT27 can do this when IO and site is complete - I don't know the language_

**Comment by kokonut27** [comment]
Okay! Thanks! Let me know where to include this, and I'll put it in :)

**Comment by transcental** [comment]
Ok. Not at all ready for it yet!

Just adding the stuff for the project board.

**Comment by kokonut27** [comment]
Okay! 👍

**Comment by transcental** [comment]
I've been thinking about the documentation, and it'd probably be better in a separate repo because then we can use a documentation framework like Sphinx(hopefully not Sphinx though).
As it'll be on the org, it'll still be linkable to this site with `/docs`

How do you want the documentation to look: - _A few examples_

| Example | Type | Runnable on GH Pages | File type | Easy to use(1-10) |
| --- | --- | --- | --- | --- |
| [Python docs](https://docs.python.org) | Sphinx | Yes | .rst | 1 |
| [Taterzens docs](https://samolego.github.io/Taterzens) | Material for MkDocs | Yes | .md | 7 |
| [Replit docs](https://docs.replit.com) | Custom | No - Replit | .md | 6 |
| [MkDocs docs](https://www.mkdocs.org/getting-started/) | MkDocs | Yes | .md | 7 |
> *1 is hardest, 10 is easiest*

I would suggest going with Material for MkDocs but you should choose.

**Comment by kokonut27** [comment]
Yeah, I think MkDocs is the best as well. Thanks for letting me know!

**Comment by transcental** [comment]
I'll create a basic boilerplate in a repl then you can add whatever should go there when it's ready.

Unless you want to set it up?
